kik_unofficial/datatypes/xmpp/auth_stanza.py

```diff
@@ -5,58 +5,58 @@
 import logging
 import pyDes
 import rsa
-import time
 
+from kik_unofficial.datatypes.xmpp.base_elements import XMPPElement
 from kik_unofficial.utilities.cryptographic_utilities import CryptographicUtils
-
+from kik_unofficial.utilities.kik_server_clock import KikServerClock
 
 log = logging.getLogger(__name__)
 identifierHex = "30820122300d06092a864886f70d01010105000382010f00"
 
 
-class AuthStanza():
+class AuthStanza(XMPPElement):
     client = None
-    keyBytes: bytes = None
-    secretKey: bytes = None
-    public_key: rsa.key.PublicKey = None
-    private_key: rsa.key.PrivateKey = None
+    key_bytes: bytes = None
+    secret_key: bytes = None
+    public_key: bytes = None
```

Comment:
Maybe call this `rsa_public_key` and `rsa_private_key` to be clear

Reply:
Resolved in 27f84e81a40d471dc14dcc85fb46c7348bc7aadc